src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1988,7 +1988,8 @@ void GraphOptimizer::FusePerformedAsScaleShiftAndFakeQuantize(Graph &graph) {
 
         const auto &outputShape = child->getOutputShapeAtPort(0);
         VectorDims outputDims = outputShape.getDims();
-        const auto channelPos = parent->getParentEdgeAt(0)->getParent()->getFusingAxis();
+        const auto nonBiasPort = (parent->getParentEdgeAt(0)->getParent()->isConstant() ? 1 : 0);
```

Comment:
If I remember it correctly, it was not a bias on this port, but Multiply or something.
We could just name it nonConstPort or maybe activationPort

Reply:
Yes, the final problem is wrong channel dim, provided by const node, but more precisely, we try to avoid calling of channel-dim from Add-containing subgraph from Mul (`parent`) node, where Add considered as bias in Add-Mul chain. As I remember, the whole problem caused by previous Eltwise fusing, which resulted in undefined Mul-port for fused Subgraph input. To highlight this rootcause, I named it nonBias port, because nonConst looks not obvious. Anyway, both variants are right. If it necessary, I can change it.

Reply:
By the way, we don't know exactly, why channel-dim is called from `parent->parent` node, which finally provoked the problem, so, some appearance of workaround is inevitable at this moment.

Reply:
More precisely would be to name it `weightPort`, as in theory we call A in `A*X + B` expression, but it would be too ambiguous here. So, we get `non-bias`, by the way of excepting.

Reply:
> I meant that name nonBiasPort means that the opposite one is actually bias port (the port with Add) which is not correct if I am not mistaken. As for reason of paret->parent I think it is because this transformation knows that the resulting ScaleShift will be eventually fused into some other node by another transformation and to have this whole fusing chain mathematically correct it has to ensure that both ScaleShift and Multiply operate on the same channel as the parent node which they are going to fused into (MatMul in this case)

It's not true. Second port is Mul. The node itself is Add.

Reply:
> I meant that name nonBiasPort means that the opposite one is actually bias port (the port with Add) which is not correct if I am not mistaken. As for reason of paret->parent I think it is because this transformation knows that the resulting ScaleShift will be eventually fused into some other node by another transformation and to have this whole fusing chain mathematically correct it has to ensure that both ScaleShift and Multiply operate on the same channel as the parent node which they are going to fused into (MatMul in this case)

This explanation of `parent->parent` logic looks reasonable in theory, but still don't explain, why we cannot rely on current channel dim. Even by the reason to guarantee the after-fusing result dim, it looks as workaround. Anyway we can only suggest the reason.

Reply:
@EgorDuplensky ?